cmake/pythonFormat.cmake

```
@@ -0,0 +1,33 @@
+# Additional target to run python linters and formatters on python scripts
+#
+# Requires black/flake8 to be available in the environment
+
+
+# Get all our Python files
+file(GLOB_RECURSE ALL_PYTHON_FILES ${PROJECT_SOURCE_DIR}/*.py)
```

```suggestion
file(GLOB_RECURSE ALL_PYTHON_FILES ${PROJECT_SOURCE_DIR}/python/*.py)
```
only pick up python files (that are not installled)